### ISSUE TITLE
The PLAYLIST and USERS buttons don't work

### ISSUE THREAD

**Issue by mik13ST** [RobertLemmens/youtube-sync#5]
I noticed the PLAYLIST button doesn't display the playlist popup window anymore. The same goes for the USERS button. It would suggest something is wrong with the popups. This is broken in the currently deployed version.

I didn't look for the commit that could have caused this, because I have never ran this app on my machine, because I don't know Scala workflow and tooling.

**Comment by RobertLemmens** [comment]
Thanks for reporting, that seemed to happen because the server i run stuff on now has https ingress and jquery was loaded through a http CDN so got blocked. I pushed a new version with updated index that uses a https CDN so it should be fixed (5f63154)

**Comment by mik13ST** [comment]
I noticed the site uses HTTPS now, which is great.

Oh, think I have never faced this issue ever. Interesting.

I can confirm it's fixed on current master (06ab0e6). Closing.

Why use jQuery though? It's thick (84kB), not needed and deprecated. It seems like you only use it for the modal initialization where it's not required (see https://materializecss.com/modals.html#initialization). And the materialize library/framework doesn't require jQuery either.
